src/org/traccar/protocol/RitiProtocolDecoder.java

```diff
@@ -66,15 +66,15 @@ protected Object decode(
         position.setDeviceId(deviceSession.getDeviceId());
 
         position.set("mode", buf.readUnsignedByte());
-        position.set("command", buf.readUnsignedByte());
-        position.set(Position.KEY_POWER, buf.readUnsignedShort());
+        position.set(Position.KEY_COMMAND, buf.readUnsignedByte());
+        position.set(Position.KEY_POWER, buf.readUnsignedShort());  // mV
 
-        buf.skipBytes(5);
-        buf.readUnsignedShort();
-        buf.readUnsignedShort();
+        buf.skipBytes(5);  // status
+        buf.readUnsignedShort();  // idleCount
+        buf.readUnsignedShort();  // idleTime in seconds
 
-        position.set(Position.KEY_DISTANCE, buf.readUnsignedInt());
-        position.set(Position.KEY_TRIP_ODOMETER, buf.readUnsignedInt());
+        position.set(Position.KEY_DISTANCE, buf.readUnsignedInt());  // meters
```

Comment:
Position.KEY_DISTANCE is the appropriate key.  In the RitiProtocol documentation, it states:

> **3.2.9 Mileage(4 bytes, LSB - MSB)**
> Moving distance, calculates the distance between current data, and latest
> previous data. Unit: meter
> **3.2.10 Total Mileage(4 bytes, LSB - MSB)**
> Total moving distance, refreshes the calculations when ACC is ON at each time.
> Unit: meter

The comment was removed in a previous commit.